packages/h5p-server/src/ContentStorer.ts

```diff
@@ -236,12 +236,18 @@ export default class ContentStorer {
                 )
             );
 
+        const filepathTonewFilenameMap = new Map<string, string>();
```

Comment:
should probably be filepathTo**N**ewFilenameMap

Reply:
Good catch. I'll also go ahead and change filepath to file**P**ath for consistency with other places.